mm2src/coins/eth/eth_tests.rs

```diff
@@ -28,6 +28,8 @@ cfg_native!(
 const GAS_PRICE_PERCENT: u64 = 10;
 const MATIC_CHAIN_ID: u64 = 137;
 
+const ETH: &str = "ETH";
+
```

Comment:
this isn't used outside of `fn test_get_enabled_erc20_by_contract_and_platform()` please move it inside